map/map_tf_generator/src/map_tf_generator_node.cpp

```diff
@@ -54,16 +58,16 @@ class MapTFGeneratorNode : public rclcpp::Node
     PointCloud clouds;
     pcl::fromROSMsg<pcl::PointXYZ>(*clouds_ros, clouds);
 
-    const unsigned int sum = clouds.points.size();
+    const std::vector<int> indices = UniformRandom(clouds.size(), N_SAMPLES);
     double coordinate[3] = {0, 0, 0};
-    for (unsigned int i = 0; i < sum; i++) {
+    for (const int i : indices) {
```

Comment:
```suggestion
    for (const auto i : indices) {
```

Reply:
```suggestion
    for (const auto & i : indices) {
```

Reply:
Actually, I believe you don't need `&` for primitive types (in this case, `int` or `size_t`).

Reply:
ah, sorry, if it's primitive type, you are rigth!

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/890/commits/9ef92be63986d219e34cc0aeb6c628746be5296f